guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>It would be better if you can follow the convention of citing a web page (different from a scientific paper). </p><p>For example, [author name, article title, url, accessed date] -&gt; Howard Jeremy. Fastai - Dynamic U-Net.&nbsp;<a href="https://www.youtube.com/watch?v=0frKXR-2PBY" target="_blank">https://www.youtube.com/watch?v=0frKXR-2PBY</a>. Accessed 2 September 2019.</p><p>You can refer to <a href="https://owl.purdue.edu/owl/research_and_citation/mla_style/mla_formatting_and_style_guide/mla_works_cited_electronic_sources.html" target="_blank">https://owl.purdue.edu/owl/research_and_citation/mla_style/mla_formatting_and_style_guide/mla_works_cited_electronic_sources.html</a> for more detail.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done. </p>